packages/components/inputs/rich-text-input/README.md

```diff
@@ -1,49 +1,95 @@
+<!-- THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY. -->
+<!-- This file is created by the `yarn generate-readme` script. -->
+
 # RichTextInput
 
 ## Description
 
-A controlled rich text input component for rich text with validation
-states.
+A controlled rich text input component for rich text with validation states.
+
+## Installation
+
+```
+yarn add @commercetools-uikit/rich-text-input
+```
+
+```
+npm --save install @commercetools-uikit/rich-text-input
+```
+
+Additionally install the peer dependencies (if not present)
+
+```
+yarn add react react-intl
+```
+
+```
+npm --save install react react-intl
+```
 
 ## Usage
 
-```js
+```jsx
+import React from 'react';
```

Comment:
I don’t know how the other files were committed but `eslint` prevented me from committing without adding this line